meta-fobnail-distro/recipes-fobnail/fobnail-attester/fobnail-attester_git.bb

```diff
@@ -0,0 +1,28 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ff626c483f6f10b69ee67274daf0775"
+
+SRC_URI = " \
+    git://github.com/fobnail/fobnail-attester;protocol=https;branch=main \
+    file://0001-Makefile-fix-to-build-in-Yocto.patch \
+    file://fobnail_usb.network \
+"
+
+PROVIDES:${PN} += "fobnail-attester"
```

Comment:
@kotylamichal PN is already in PROVIDES by default https://docs.yoctoproject.org/ref-manual/variables.html#term-PROVIDES

Reply:
removed